src/app/pages/security/security.page.ts

```diff
@@ -2,6 +2,9 @@ import { Component, OnInit } from '@angular/core';
 import { DeviceCheckResult, DeviceCheckType, SecurityService } from '@aerogear/security';
 import { Dialogs } from '@ionic-native/dialogs/ngx';
 import { Platform } from '@ionic/angular';
+import { ConfigurationService, MetricsService } from '@aerogear/core';
+
+const config = require('../../../mobile-services.js');
```

Comment:
We have a service (OpenshiftService) to interact with this file.
Direct usage may not be recomended.

Reply:
👍 done. Mind giving another look?